util/util_test.go

```diff
@@ -129,3 +131,34 @@ func TestMultiWordReplace(t *testing.T) {
 		})
 	}
 }
+
+func TestOutputResultsJSON(t *testing.T) {
+	outcfg := &OutputConfig{
+		OutputFile: "/tmp/bench-common-tst1.jout",
+		JSONFmt:    true,
+	}
+	controls := &check.Controls{}
+	summary := check.Summary{
+		Pass: 10,
+		Fail: 0,
+		Warn: 2,
+	}
+	err := OutputResults(controls, summary, outcfg)
```

Comment:
Bit of an optional extra but it would be nice to test that the json output is as expected

Reply:
Thinking twice about this, the test is highlighting the problem of a filename being passed in instead of `io.Writer`